src/ballet/sbpf/fuzz_sbpf_loader.c

```diff
@@ -52,7 +52,7 @@ LLVMFuzzerTestOneInput( uchar const * data,
     fd_sbpf_syscalls_insert( syscalls, *x );
 
   /* Load program */
-  int res = fd_sbpf_program_load( prog, data, size, syscalls );
+  int res = fd_sbpf_program_load( prog, data, size, syscalls, true );
```

Comment:
We should add `deploy_mode` to the protobuf so that the fuzzer can check both configurations.

Reply:
Good point!

Reply:
This particular fuzzer is no longer used (and should be removed at some point), but I will work on this in solfuzz

Reply:
Yep sorry, I was referring to solfuzz!